Dockerfile

```
@@ -0,0 +1,9 @@
+FROM python:3.9
+
+RUN pip install fastapi uvicorn
+
+EXPOSE 80
+
+COPY ./app /app
```

Let's move the API module into the agent_testing folder (the core package for this project) and copy that into the dockerfile instead of app :slightly_smiling_face: 
```suggestion
COPY ./agent_testing /agent_testing
```